src/ExtInterface/ADRIA/Domain.jl

```diff
@@ -114,6 +114,7 @@ function Domain(name::String, dpkg_path::String, rcp::String, timeframe::Vector,
 
     # Filter out missing entries
     site_data = site_data[coalesce.(in.(conn_ids, [site_conn.site_ids]), false), :]
+    site_data.k .= site_data.k / 100.0  # Make `k` non-dimensional (provided as a percent)
```

Comment:
It doesn't really matter but what's the reason for moving division by 100 outside of the function?

Reply:
The reason is to ensure the value in `k` is always loaded as $\in [0, 1]\$ regardless of the source of the data for consistency.

If it is inside `location_k`, it needs to be divided by 100 every time, and we need separate functions to handle sources that provide it as [0, 1] vs [0, 100], or some future hypothetical source that provides absolute units (m^2, km^2, hectares, etc).

Best to do it once in the relevant `load_domain()`.

Reply:
Thanks, makes sense